statsmodels/genmod/families/links.py

```diff
@@ -586,6 +586,149 @@ class log(Log):
     pass
 
 
+class LogC(Link):
+    """
+    The log-complement transform
+
+    Notes
+    -----
+    call and derivative call a private method _clean to trim the data by
+    machine epsilon so that p is in (0,1). logc is an alias of LogC.
+    """
+
+    def _clean(self, x):
+        return np.clip(x, FLOAT_EPS, np.inf)
```

Comment:
this `clean` method is applied to probability `p` in the methods below.
should clip to (0, 1) and not (0, inf)

however, the restriction for the link function to be valid is 1-x>0 or x < 1

Reply:
Agreed. Good catch.

Reply:
This is fixed. Commit squashed.